server/server.go

```diff
@@ -40,8 +42,17 @@ type ListenOpts struct {
 	// Override legacy behavior where server logs written to the application's logrus object
 	// were dropped unless logrus was set to debug-level (such as by launching steve with '--debug').
 	// Setting this to true results in server logs appearing at an ERROR level.
-	DisplayServerLogs bool
+	DisplayServerLogs       bool
+	IgnoreTLSHandshakeError bool
 }
+type tlsHandshakeErrorWriter struct {
+	defaultWriter io.Writer
+	debugWriter   io.Writer
+}
```

Comment:
Do we really need a whole new type, another bool flag, and a bunch of tests just to do a `strings.Contains` on the error message?